pkgs/snuba/snuba.patch

```diff
@@ -0,0 +1,94 @@
+diff --git a/requirements.txt b/requirements.txt
+index 0ef64e20..69ed0a8f 100644
+--- a/requirements.txt
++++ b/requirements.txt
+@@ -1,11 +1,10 @@
+-appnope==0.1.0
++appnope==0.1.0; sys_platform == 'darwin'
+ argh==0.26.2
+-astroid==1.6.5
++astroid
```

Comment:
It could be easier just to use sed to strip off all version numbers from requirements.txt.

Reply:
Fixed.